templates/test/infopanel.html.ep

```diff
@@ -81,7 +81,10 @@
                 % }
                 % if ($worker) {
                     <div>
-                        Assigned worker: <span title="VNC port: <%= 90 + $worker->instance %>"><%= $worker->to_string %></span>
+                        Assigned worker:
+                        <span title="VNC port: <%= 90 + $worker->instance %>">
+                            %= link_to $worker->to_string => url_for('admin_worker_show', worker_id => $worker->id)
```

Comment:
I can double check, but I'm sure that the test wasn't running when I tested.

If the user is no operator he can not access the page obviously. However, this will change when my other PR gets finally merged.